ci-scripts/publish.sh

```diff
@@ -16,4 +16,8 @@ else
     git push --verbose --follow-tags "https://$GH_TOKEN@github.com/$TRAVIS_REPO_SLUG" "$TRAVIS_BRANCH" > /dev/null 2>&1;
 
     npm publish --tag prerelease
+
+    # deploy github pages
+    npm run predeploy
+    npm run deploy
```

Comment:
@droshev were there any other steps than this to deploy to gh-pages?

Reply:
@jbadan no, that's it